tests/operator-subscription.yaml

```diff
@@ -4,6 +4,14 @@ metadata:
   name: my-operator-subscription
   namespace: operator
 spec:
+  config:
+    env:
+    - name: HTTP_PROXY
+      value: <http_proxy>
+    - name: HTTPS_PROXY
+      value: <https_proxy>
+    - name: NO_PROXY
+      value: <no_proxy>
   channel: alpha
   name: ovms-operator
   startingCSV: openvino-operator.v1.0.0
```

Comment:
Could you also update this line to v1.1.0 ?

Reply:
Done.